### PR TITLE
Poddy as podsource

### PR DESCRIPTION
**Pull request by gsantner**
- Use poddy (see diaspora-android-extras) as source for pods
- Update app to use new DiasporaPod
- Update PodSelectionFragment

### CODE REVIEW

File: app/src/main/java/com/github/dfa/diaspora_android/data/DiasporaPodList.java [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+
+    public Integer getTrackAddedIndexStart() {
+        return trackAddedIndexStart;
+    }
+
+    public List<Integer> getTrackUpdatedIndexes() {
+        return trackUpdatedIndexes;
+    }
+
+    /*      ██████╗  ██████╗ ██████╗
```
Comment (vanitasvitae) [inline app/src/main/java/com/github/dfa/diaspora_android/data/DiasporaPodList.java:173]: This is indented :D

File: app/src/main/java/com/github/dfa/diaspora_android/task/GetPodsService.java [hunk 2]
```diff
@@ -53,75 +57,48 @@ public int onStartCommand(Intent intent, int flags, int startId) {
...
-
-                StringBuilder builder = new StringBuilder();
-                //HttpClient client = new DefaultHttpClient();
-                List<String> list = null;
-                HttpsURLConnection connection;
-                InputStream inStream;
+            protected DiasporaPodList doInBackground(Void... params) {
+                StringBuilder sb = new StringBuilder();
+                BufferedReader br = null;
                 try {
```
Comment (vanitasvitae) [inline app/src/main/java/com/github/dfa/diaspora_android/task/GetPodsService.java:61]: Should we make the online query to githubs servers optional/toggleable?

Comment (gsantner) [reply app/src/main/java/com/github/dfa/diaspora_android/task/GetPodsService.java:61]: We can if wanted

File: app/src/main/java/com/github/dfa/diaspora_android/data/DiasporaPodList.java (hunk 1 above)
Comment (gsantner) [reply app/src/main/java/com/github/dfa/diaspora_android/data/DiasporaPodList.java:173]: was

**Pull request merged**